drivers/storage/portworx/components.go

```diff
@@ -299,16 +299,41 @@ func (p *portworx) unsetInstallParams() {
 
 func createVolumePlacementStrategyCRD() error {
 	logrus.Debugf("Creating VolumePlacementStrategy CRD")
+
 	resource := k8s.CustomResource{
-		Name:       "volumeplacementstrategy",
-		Plural:     "volumeplacementstrategies",
-		Group:      "portworx.io",
-		Version:    "v1beta1",
-		Scope:      apiextensionsv1beta1.ClusterScoped,
-		Kind:       "VolumePlacementStrategy",
-		ShortNames: []string{"vps", "vp"},
-	}
-	err := k8s.Instance().CreateCRD(resource)
+		Plural: "volumeplacementstrategies",
+		Group:  "portworx.io",
+	}
+
+	crd := &apiextensionsv1beta1.CustomResourceDefinition{
```

Comment:
This shouldn't be done by the operator. This needs to be maintained by Portworx since there is a dependency on which version of PX can use which version of this CRD.

Reply:
We discussed offline, we will move this to Portworx eventually. For now keeping it operator until Portworx registers it's own CRD.